src/DIRAC/Resources/Computing/ARCComputingElement.py

```diff
@@ -79,9 +95,80 @@ def __init__(self, ceUniqueID):
 
     #############################################################################
 
+    def _isThereRestInterface(self):
+        # The name is the documentation. Importantly, if all is well the following variables are set
+        # - and a few others on which they (especially self.s) depend.
+        # self.rest    : Static variable to say if the REST interface is available and ready
+        # self.base_url: The basic URL for the REST interface
+        # self.s       : The python "requests" module session
+        # self.headers : The format of information we prefer : json of course
+        #
+        self.log.debug("Testing if the REST interface is available for %s" % self.ceHost)
+
+        # Is this needed? Test if the CE is supposed to be an ARC CE.
+        result = getCESiteMapping(self.ceHost)
+        if not result["OK"] or not result["Value"]:
+            self.log.error("Unknown CE ...")
+            return
+        self.site = result["Value"][self.ceHost]
+        grid = self.site.split(".")[0]
+        ceString = "Resources/Sites/%s/%s/CEs/%s/CEType" % (grid, self.site, self.ceHost)
+        print(ceString)
+        result = gConfig.getValue(ceString, defaultValue="")
+        if result != "ARC":
+            self.log.info("This ce (%s) is not an ARC CE. It is : %s. Returning." % (self.ceHost, result))
+            self.rest = False
+            return
+
+        # Is the ARC endpoint published / available. It is also a check on the CE health.
+        self.log.debug("First check if the CE %s publishes its REST endpoint" % self.ceHost)
+        command = """ ldapsearch -x -LLL -h %s:2135 -b 'o=glue' \
+                  GLUE2EndpointInterfaceName=org.nordugrid.arcrest GLUE2EndpointURL \
+                  | grep GLUE2EndpointURL | awk -F ": " '{print $2}' """
+        status = shellCall(1, command % self.ceHost)
+        service_url = ""
+        if status["OK"]:
+            service_url = status["Value"][1].strip()
+        if len(service_url) < 5:  # CE is probably down. Wait for next iteration to test again
+            self.log.debug("CE %s probably down. Please contact the site admin." % self.ceHost)
+            self.rest = False
+            return
+
+        # We now have a pointer (service_url) to the REST interface for this CE
+        restVersion = "1.0"
+        self.base_url = service_url + "/rest/" + restVersion + "/"
+        command = "info?schema=glue2"
+        query = self.base_url + command
+
+        # Set up the request framework
+        self.s = requests.Session()
+        result = self._prepareProxy()
+        if not result["OK"]:  # Probably everything is going to fail?
+            self.log.info("No proxy found -  Probably just no jobs to run for this CE?")
+            self.log.info("Setting REST interface false.")
+            self.rest = False
+            return
+        self.s.cert = os.environ["X509_USER_PROXY"]
+        self.s.verify = "/etc/grid-security/certificates/"
```

Comment:
while this is standard-ish, you want to use
```suggestion
        self.s.verify = os.environ["X509_CERT_DIR"]
```